pkg/apiserver/archive/archive.go

```diff
@@ -278,38 +279,62 @@ func (s *Service) report(c *gin.Context) {
 // @Description Delete the specified archived experiment.
 // @Tags archives
 // @Produce json
-// @Param uid path string true "uid"
+// @Param uid path string false "uid"
```

Comment:
That is so weird. Can we delete this path later?

Reply:
Or make it `//@Router /archives [delete]` only, and split `//@Router /archives/{uid} [delete]` to another api

Reply:
Agree to split them.